integration-test/src/test/resources/test-ds.xml

```diff
@@ -30,7 +30,7 @@
       <!-- Properties for Hibernate -->
       <property name="hibernate.dialect" value="org.hibernate.dialect.HSQLDialect"/>
       <property name="hibernate.hbm2ddl.auto" value="create-drop"/>
-      <property name="hibernate.show_sql" value="true"/>
+      <property name="hibernate.show_sql" value="false"/>
```

Comment:
Why changing it?

Reply:
This was set to true by me some days ago, I am just reverting it back